cpp/ql/test/library-tests/dataflow/fields/A.cpp

```diff
@@ -40,21 +40,21 @@ class A
     cc.insert(nullptr);
     ct.insert(new C());
     sink(&cc); // no flow
-    sink(&ct); // $ast $f-:ir
+    sink(&ct); // $ ast MISSING: ir
```

Comment:
Is the space between the `$` and the tag (`ast`) optional?

Reply:
Yes, it's optional. I've only added the space because I found it more pretty. The `$` symbol now means "this comment is an inline-expectation test", and it's no longer attached to a tag.